CITATION.cff

```diff
@@ -0,0 +1,166 @@
+    email: markiewicz@stanford.edu
+    orcid: https://orcid.org/0000-0002-6533-164X
+    affiliation: Stanford University
+
+  - family-names: Bansal
+    given-names: Shashank
+    website: https://shashankbansal6.github.io/
+    email: shashankbansal56@gmail.com
+
+  - family-names: Blair
+    given-names: Ross W. 
+    website: https://github.com/rwblair
+
+  - given-names: Katherine
+    family-names: Bottenhorn
+
+  - family-names: Chen
+    given-names: Gang
+    website: https://github.com/afni-gangc
+    orcid: https://orcid.org/0000-0002-2960-089X
+    affiliation:
+      National Institutes of Health, Scientific and Statistical Computing Core,
+      NIMH, Bethesda, MD 20892, USA
+    email: gangchen@mail.nih.gov
+
+  - family-names: Dafflon
+    given-names: Jessica
+    website: https://jessicadafflon.github.io/
+
+  - family-names: Nielson
+    given-names: Dylan
+    affiliation:
+      Machine Learning Team at the National Institute of Mental Health
+    website: https://github.com/Shotgunosine
+
+  - family-names: Esteban
+    given-names: Oscar
+    website: http://www.axonlab.org
+    orcid: https://orcid.org/0000-0001-8435-6191
+    affiliation:
+      Department of Radiology, Lausanne University Hospital, Lausanne, CH-1011,
+      Switzerland
+    email: phd@oscaresteban.es
+
+  - family-names: Gau
+    given-names: Rémi
+    affiliation: Université catholique de Louvain
+    email: remi.gau@uclouvain.be
+    orcid: "https://orcid.org/0000-0002-1535-9767"
+
+  - family-names: Kent
+    given-names: James
+    affiliation:
+      Psychoinformatics Laboratory at the University of Texas at Austin
+    website: https://jdkent.github.io/
+    email: jamesdkent21@gmail.com
+
+  - given-names: Camille
+    family-names: Maumet
+    website: https://camillemaumet.com/
+    orcid: https://orcid.org/0000-0002-6290-553X
+    affiliation: Inria, Univ Rennes, CNRS, Inserm, Rennes, 35042, France
+
+  - family-names: Mumford
+    given-names: Jeanette
+    website: https://github.com/jmumford
+    email: jeanette.mumford@gmail.com
+
+  - family-names: Narayan
+    given-names: Manjari
+    email: manjari@alumni.rice.edu
+
+  - family-names: Nichols
+    given-names: Thomas E. 
+    website: http://nisox.org
+    orcid: https://orcid.org/0000-0002-4516-5103
+    affiliation:
+      "University of Oxford, Big Data Institute, Oxford, OX3 7LF, UK "
+    email: thomas.nichols@BDI.OX.AC.UK
+
+  - given-names: Hernando
+    family-names: Ombao
+    email: hernando.ombao@kaust.edu.sa
+
+  - family-names: Guiomar Niso Galán
+    given-names: Julia
+    website: http://guiomarniso.com
+    orcid: https://orcid.org/0000-0001-5872-8924
+    affiliation: Indiana University
+    email: guiomar.niso@ctb.upm.es
+
+  - family-names: Poldrack
+    given-names: Russell A. 
+    website: http://www.poldracklab.org
+    orcid: https://orcid.org/0000-0001-6755-0259
+    affiliation: Stanford University, Stanford, CA 94305 USA
+    email: russpold@stanford.edu
```

Comment:
@poldrack This will be used to credit you as a co-author of the BIDS Stats Models specification. Please confirm that you would like to be listed as an author and update the metadata as you see fit.